meta/app.grayjay.Desktop.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=GrayJay Desktop
+Comment=Multi-platform video feed aggregator
+Categories=AudioVideo;Player;
+
+Icon=app.grayjay.Desktop
+Exec=/app/grayjay/Grayjay
```

Comment:
exec is NEVER a full path.

Reply:
ah whoops, this was leftover from some testing. the linter was able to catch it in the main manifest, but not in here. my bad.